hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticLoadBalancerRegionReplica.java

```diff
@@ -49,6 +51,41 @@
   public static final HBaseClassTestRule CLASS_RULE =
       HBaseClassTestRule.forClass(TestStochasticLoadBalancerRegionReplica.class);
 
+  // Mapping of locality test -> expected locality
+  private float[] expectedLocalities = {1.0f, 0.5f};
+
+  /**
+   * Data set for testLocalityCost:
```

Comment:
Perhaps there's a more legible representation of cluster locality than these jagged arrays?

Reply:
The format of input array is required by MockCluster, will see if I can add more comments for easy reading.

Reply:
Yeah looks like the existing system is... wow.